docs/advanced/interface_types.mdx

```diff
@@ -73,3 +73,11 @@ Counts the number of embeddings (chunks) in the database.
 print(app.count())
 # returns: 481
 ```
```

Comment:
Hey @Dev-Khant : i though about the documentation one.
Planning to do a separate spring to improve the docs.

Can you remove this for now? this doesn't heading doesn't fit well in the page and right now before creating a new page, want to make a note of all the improvements that needs to be done in the documentation.

Reply:
Alright